### ISSUE TITLE
Regarding Figure 1 in the paper, the issue of parameter freezing in the MedSAM model.

### ISSUE THREAD

**Issue by Zehui-Lin** [xianlin7/SAMUS#22]
In Figure 1 of the paper, the ViT Encoder in the MedSAM model is frozen and not fine-tuned. However, according to the original text of MedSAM, it states, "The model was initialized with the pre-trained SAM model with the ViT-Base model. We fixed the prompt encoder since it can already encode the bounding box prompt. All the trainable parameters in the image encoder and mask decoder were updated during training. Specifically, the number of trainable parameters for the image encoder and mask decoder are 89,670,912 and 4,058,340, respectively." Therefore, the MedSAM model should only freeze the prompt encoder while fine-tuning the ViT Encoder and mask decoder.
